QuickStart.md

```diff
@@ -0,0 +1,219 @@
+      ResponseSpecification expectedResponse) {
+    RequestSpecBuilder reqBuilder = getDefaultReqSpecBuilder();
+    reqBuilder.setBody(object.getObjectMetadata());
+
+    Response response = fireRestCall("SampleService", objectPath, reqBuilder,
+          expectedResponse, method.POST);
+
+    return response;
+  }
+}
+```
+
+Ok, now we have our config information setup and our basic object and controller code.  Let's compile it into a library.
+
+From either your IDE or the location of your pom in terminal/command line window, run the standard maven install command:
+``` mvn clean install ```
+
+## Test Project and Code
+
+Now we have a distributable library that you can post on your internal artifact repository (once you get it into a shape you like, for now we will use your locally compiled version).
+
+Let's create a project to write our tests.  In your IDE create a new Java Project with a groupId and artifactId different from your library, something like com.mycompany.mywebservice.automation /  webservice-tests
+
+In your pom, add a dependency for the library you just created:
+
+```
+	<dependency>
+      <groupId>com.mycompany.mywebservice</groupId>
+      <artifactId>webservice-automation-library</artifactId>
+      <version>0.0.1-SNAPSHOT</version>
+    </dependency>
+```
+
+Add the following package/class in the test source of your project
+
+src/test/java
+	- com.mycompany.mywebservice.tests
+		- Basic\_Test\_IT.java
+		
+Open Basic\_Test\_IT.java and add code until it looks like the following:
+
+```
+import java.util.UUID;
+import org.testng.annotations.Test;
+import com.adobe.ride.core.types.ExpectedResponse;
+import com.mycompany.mywebservice.core.MyWebServiceController;
+import com.mycompany.mywebservice.objects.SampleServiceObject1;
+
+public class BasicTest_IT {
+
+  @Test(groups = {"smoke", "acceptance"})
+  public void testAuthenticatedCalltoServer() {
+    String itemName = UUID.randomUUID().toString();
+    // Create Object and dynamically generate data from schema
+    SampleServiceObject1 testObject = new SampleServiceObject1(itemName, false);
+
+    // Send Object To Server. 
+    MyWebServiceController.firePOSTCall(testObject.getName, testObject,
+        ExpectedResponse.CREATED_RESPONSE);
+  }
+}
+```
+
+Let's run the test, and specify which environment we want to target in the config definitions from the terminal:
+
+``` mvn clean verify -Dtarget=stage01 ```
+
+When it's run your REST call should be generated and logged, which you'll see in the console, and assuming your webservice exists in the way it is specified in the configs, you should see a response come back
+
+## Final Thoughts
+
+With a little bit of upfront work, you can see that you can have an extremely useable and flexible library to generate tests against your webservice in just a few lines of code.  From there you can delve into leverage all of the resources available in Ride to really provide some comfort that your service is being fully tested.
```

Comment:
awkward sentence: 
From there you can delve into leverage all of the resources available in Ride to really provide some comfort that your service is being fully tested.

Reply:
Fixed.